### PR TITLE
RHELSQ-4656: Conditional skip for container embedding test

### PR DESCRIPTION
**Pull request by atodorov**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: test/cases/image_tests.sh
Comment (achilleas-k) [inline test/cases/image_tests.sh:64]: Not a big issue, but the actual new options are `manifest-lists` (for skopeo inputs) and the new `skopeo-index` source.
It might save someone some trouble in the future if they went looking for the reason for this skip.

**Review by achilleas-k — APPROVED** [review]
LGTM.  Could clarify the comment in the image tests script filtering, but not a requirement.

**Comment by achilleas-k** [comment]
A small head's up: I made a change to the same test script (container embedding) in https://github.com/osbuild/osbuild-composer/pull/3398, so if that one gets merged first, we'll need to resolve the conflict here.  Happy to add the same condition if this PR gets merged ahead of it.

File: test/cases/image_tests.sh
Comment (atodorov) [reply test/cases/image_tests.sh:64]: Updated.

File: test/cases/container-embedding.sh
Comment (atodorov) [reply test/cases/container-embedding.sh:148]: Yes. See https://redhat-internal.slack.com/archives/C0235DZB0DT/p1681975914663989?thread_ts=1681889588.279369&cid=C0235DZB0DT

**Comment by jrusz** [comment]
I can see the packer job failing, I guess it needs https://github.com/osbuild/osbuild-composer/pull/3400 ?

**Comment by ondrejbudai** [comment]
The packer job failure is unrelated, let's merge this.

**Comment by atodorov** [comment]
> The packer job failure is unrelated, let's merge this.

Thanks. I was waiting for the nightly CI job to complete before we can merge. https://gitlab.com/redhat/services/products/image-builder/ci/osbuild-composer/-/pipelines/843026534

**Pull request merged**
